src/diagnostics/CMakeLists.txt

```diff
@@ -102,12 +102,15 @@ endif(BUILD_CRASHPAD_CLIENT)
 
 include(${PROJECT_SOURCE_DIR}/build/module.cmake)
 
-# === Install crashpad handler ===
-set(CPAD_ROOT_PATH ${CMAKE_CURRENT_LIST_DIR}/crashpad_handler)
-if (OS_IS_LIN)
-    install(PROGRAMS ${CPAD_ROOT_PATH}/linux/x86-64/crashpad_handler DESTINATION bin)
-elseif(OS_IS_WIN)
-    install(PROGRAMS ${CPAD_ROOT_PATH}/windows/x86-64/crashpad_handler.exe DESTINATION bin)
-elseif(OS_IS_MAC)
-    install(PROGRAMS ${CPAD_ROOT_PATH}/macos/x86-64/crashpad_handler DESTINATION ${Mscore_SHARE_NAME}Contents/MacOS)
-endif()
+#if (BUILD_CRASHPAD_CLIENT)
+#    # === Install crashpad handler ===
+#    set(CPAD_ROOT_PATH ${CMAKE_CURRENT_LIST_DIR}/crashpad_handler)
+#    if (OS_IS_LIN)
+#        install(PROGRAMS ${CPAD_ROOT_PATH}/linux/x86-64/crashpad_handler DESTINATION bin)
+#    elseif(OS_IS_WIN)
+#        install(PROGRAMS ${CPAD_ROOT_PATH}/windows/x86-64/crashpad_handler.exe DESTINATION bin)
+#    elseif(OS_IS_MAC)
+#        install(PROGRAMS ${CPAD_ROOT_PATH}/macos/x86-64/crashpad_handler DESTINATION ${Mscore_SHARE_NAME}Contents/MacOS)
+#    endif()
+#endif(BUILD_CRASHPAD_CLIENT)
+
```

Comment:
Hmm. I guess they should get installed, but build first?

Reply:
Yes, not sure if it builds but install doesn't find the built binaries. Or if they aren't built at all.

Reply:
But that's a problem than needs to get sorted first

Reply:
Yes, meanwhile I'll convert this PR to a draft.

Reply:
crashpad_handler prebuild binaries, without which crash dumps will not work

Reply:
Ok, so you can't build the crashpad-handler from sources (easily), and the crashpad handler will do network activity (https://chromium.googlesource.com/crashpad/crashpad/+/HEAD/handler/crashpad_handler.md).

I'm updating the PR to remove the prebuilt binaries and turn the options off by default.